src/neuroglancer/segmentation_display_state/base.ts

```diff
@@ -31,6 +31,10 @@ export interface VisibleSegmentsState {
   temporarySegmentEquivalences: SharedDisjointUint64Sets;
   useTemporaryVisibleSegments: SharedWatchableValue<boolean>;
   useTemporarySegmentEquivalences: SharedWatchableValue<boolean>;
+
+  // TODO explanation, currently they cause non focus segments to be hidden in 2d, segmentation rendered transparent, with everything else darkened
+  // to bring attention to the EM for that segment
+  focusSegments: Uint64Set;
```

Comment:
I initially had this in SegmentationDisplayState since it was just used for changing the 2d rendering, but then I also decided I could use this set to prevent users from accidentally deselecting the active multicut segment while the tool is active. I do that in the graph connection visible segment listener by re-adding the segment if it is removed (and is in the focus set).